source/compiler-core/slang-dxc-compiler.cpp

```diff
@@ -611,6 +598,14 @@ SlangResult DXCDownstreamCompiler::getVersionString(slang::IBlob** outVersionStr
             versionString.append(major);
             versionString.append(".");
             versionString.append(minor);
+
+            char* commitHash;
```

Comment:
It is safer to initialize this to nullptr, in case `getCommitInfo` call failed.

Reply:
I'll include this change with the shader cache index PR